doc/api/util.md

```diff
@@ -364,6 +364,64 @@ util.formatWithOptions({ colors: true }, 'See object %O', { foo: 42 });
 // when printed to a terminal.
 ```
 
+## `util.getCallSite(frames)`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `frames` {number} Number of frames returned in the stacktrace.
+  **Default:** `10`.
+
```

Comment:
This empty line needs to be removed. From what I see, all pieces of the method signature should be grouped together.